db/sql/sql.go

```diff
@@ -528,17 +538,28 @@ func makeStateColumns(s *schema.Schema) (cols []string) {
 	return cols
 }
 
-func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder) sq.SelectBuilder {
+func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder, aliasRelationPropertyPrefix string, aliasTableName string) sq.SelectBuilder {
```

Comment:
Why not do 

newAliasTableName = aliasTableName + "__" + property.RelationProperty

for always? if we do this, we can simplify logic.